tree/ntuple/v7/src/RDaos.cxx

```diff
@@ -203,18 +211,56 @@ int ROOT::Experimental::Detail::RDaosContainer::ReadSingleAkey(void *buffer, std
                                                                DistributionKey_t dkey, AttributeKey_t akey,
                                                                ObjClassId_t cid)
 {
+   std::vector<AttributeKey_t> akeys{akey};
```

Comment:
Also note that if `FetchUpdateArgs` constructor takes a `std::span<...>`, this (and `iovs`) might be replaced by a static allocated array (same below in `WriteSingleAkey()`).